locales/en/filters.json

```diff
@@ -53,7 +53,7 @@
 	},
 	{
 		"filter.14.name": "AdGuard Annoyances filter",
-		"filter.14.description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups."
+		"filter.14.description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. Contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances and AdGuard Widgets."
```

Comment:
Too many AdGuards there eh?

Contains Cookie Notices, Popups, Mobile App Banners, Widgets and Other Annoyances filters.

Reply:
Yes, but it also reflects their names. If we can leave it the way you suggested, fine)

Reply:
I think we can

Reply:
I did it, thanks.

Reply:
Sounds good, I have changed it @el-termikael 

> Cookies

Now it's Cookie Notices